charts/aws-mountpoint-s3-csi-driver/values.yaml

```diff
@@ -35,6 +35,12 @@ node:
     limits:
       memory: 256Mi
   tolerateAllTaints: false
+  tolerations:
```

Comment:
So with this change, if someone provides `node.tolerations`, it will override existing tolerations, which could be a breaking change I think. Imagine someone relies on default tolerations provided by the CSI Driver and with this change they will go away. Not sure how big is a deal this, one alternative could be creating a new variable like `node.defaultTolerations` for these tolerations, but then one would need to override two Helm values to tweak this configuration.

Reply:
Yes, that was also something I was thinking about.

I guess for now, creating a new value is probably the best solution without introducing a breaking change. I have now added `node.defaultTolerations` and put it in the `else` condition in `tolerateAllTaints`. For me this is the simplest solution after playing around with helm templating.